socorro/signature/rules.py

```diff
@@ -151,8 +151,6 @@ def normalize_cpp_function(self, function, line):
             if function.endswith(ref):
                 function = function[: -len(ref)].strip()
 
-        # Convert `anonymous namespace' to (anonymous namespace)
-
```

Comment:
Unrelated to the fix -- this block was just moved further down but a redundant comment was leftover.